### DISCUSSION TITLE
Copy watch history from one profile to another

### DISCUSSION THREAD

**Discussion by rg-gapa** [007revad/Plex_Server_Sync#12] [Q&A]
Is it possible to copy only watch history from one profile (managed in my case) to another (main in my case)?
I might fork docker container and volume with data attached to it, so I guess it's not required to copy from-to same server.

**Comment by 007revad** [comment]
If you have Tautulli installed you can sync watch history between user and/or between servers with:
https://github.com/blacktwin/JBOPS/blob/master/utility/sync_watch_status.py
